lib/rules/selector-disallowed-list/README.md

```diff
@@ -66,3 +66,49 @@ a
 ```css
 a[href] {}
 ```
+
+## Optional secondary options
+
+### `splitList: true | false` (default: `false`)
+
+Iterate through each individual selector in a selector list.
+
+For example, with `true`.
+
+Given:
+
+```json
+[".foo"]
+```
+
+The following pattern is considered a problem:
+
+<!-- prettier-ignore -->
+```css
+.bar, .foo {}
+```
+
+The following pattern is _not_ considered a problem:
+
+<!-- prettier-ignore -->
+```css
+.bar .foo {}
```

Comment:
[suggest] Missing closing triple-backslashes?

````suggestion
.bar .foo {}
```
````

Reply:
Eugh. My bad. Think it looks good now. 
<img width="958" alt="Screen Shot 2022-09-13 at 10 15 19 am" src="https://user-images.githubusercontent.com/12629019/189780442-bffab0f5-7592-47c9-92d6-a74d0f29b14b.png">